Step 4 missing in the installation procedure (#3713)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/installation/installation-of-a-central-server/unattended.md

Dans les deux cas, vous obtiendrez un fichier de log complet avec toutes les erreurs dans votre répertoire **tmp**, fichier nommé **unattended(date).log**.

> Pour obtenir de l'aide sur le script, utilisez la commande suivante :`bash unattended.sh -h`

4. Configurez Centreon
Connectez-vous à l'interface web de Centreon avec l'URL http://[SERVER_IP]/centreon en remplaçant [SERVER_IP] par l'adresse IP de votre serveur.
Après avoir tenté de vous connecter, suivez les instructions décrites [ici](../../web-and-post-installation).

5. Commencez à utiliser Centreon

> Suivez notre [guide de démarrage](../../getting-started/welcome.md) pour commencez à superviser.

Author: ```suggestion
Suivez notre [guide de démarrage](../../getting-started/welcome.md) pour commencer à superviser.
```

Dans les deux cas, vous obtiendrez un fichier de log complet avec toutes les erreurs dans votre répertoire **tmp**, fichier nommé **unattended(date).log**.

> Pour obtenir de l'aide sur le script, utilisez la commande suivante :`bash unattended.sh -h`

4. Configurez Centreon

Connectez-vous à l'interface web de Centreon avec l'URL `http://[SERVER_IP]/centreon` en remplaçant [SERVER_IP] par l'adresse IP de votre serveur.

> Après avoir tenté de vous connecter, suivez les instructions décrites [ici](../../web-and-post-installation/#initialization-of-the-monitoring).

Author: ```suggestion
Une fois connecté, suivez les instructions décrites [ici](../../web-and-post-installation/#initialization-of-the-monitoring).
```

Dans les deux cas, vous obtiendrez un fichier de log complet avec toutes les erreurs dans votre répertoire **tmp**, fichier nommé **unattended(date).log**.

4. Configurez Centreon
Connectez-vous à l'interface web de Centreon avec l'URL http://[SERVER_IP]/centreon en remplaçant [SERVER_IP] par l'adresse IP de votre serveur.

> Après avoir tenté de vous connecter, suivez les instructions décrites dans la [documentation Centreon](../web-and-post-installation).

Author: ```suggestion
Après avoir tenté de vous connecter, suivez les instructions décrites dans la [documentation Centreon](../../web-and-post-installation).
```

> Après avoir tenté de vous connecter, suivez les instructions décrites dans la [documentation Centreon](../../web-and-post-installation).

Author: ```suggestion
Après avoir tenté de vous connecter, suivez les instructions décrites [ici](../../web-and-post-installation).
```
